static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -0,0 +1,70 @@
+.adverts--capi {
+    .advert__image {
+        max-height: $gs-baseline * 11;
```

Comment:
Spacing is inconsistent across the files. It is small thing but it is nice to have it the same.